libs/transloco/src/lib/transloco.service.ts

```diff
@@ -685,10 +695,10 @@ export class TranslocoService implements OnDestroy {
   }
 
   ngOnDestroy() {
-    // Caretaker note: since this is the root provider, it'll be destroyed when the `NgModuleRef.destroy()` is run.
-    // Cached values capture `this`, thus leading to a circular reference and preventing the `TranslocoService` from
-    // being GC'd. This would lead to a memory leak when server-side rendering is used since the service is created
-    // and destroyed per each HTTP request, but any service is not getting GC'd.
```

Comment:
I'd like to keep the comment.
Please leave the following comment above the cache.clear:
```
// As a root provider, this service is destroyed with `NgModuleRef.destroy()`. 
// Cached values retain `this`, causing circular references that block garbage collection, 
// leading to memory leaks during server-side rendering.
```

Reply:
Left the comment.